app/assets/javascripts/listings/components/browse-listings.html.slim

```diff
@@ -92,6 +85,8 @@ section.results-section ng-if="!$ctrl.parent.openListings.length"
       .large-12.columns
         property-card listing='listing' ng-repeat="listing in $ctrl.parent.openListings"
 
+ng-include src="'listings/templates/_rental-callout.html'" ng-if="$ctrl.tenureType == 'rental' && !$ctrl.parent.hasEligibilityFilters() || $ctrl.openMatchListings.length"
```

Comment:
@callaghanc  seems like the double callback is coming from here. I'm not sure what the goal of the logic is, but maybe it should be $ctrl.tenureType == 'rental' && (!$ctrl.parent.hasEligibilityFilters() || $ctrl.openMatchListings.length) ?

Reply:
yes that makes sense, i'll push that update